pbj-core/pbj-grpc-helidon/src/main/java/com/hedera/pbj/grpc/helidon/PbjConfigBlueprint.java

```diff
@@ -0,0 +1,54 @@
+package com.hedera.pbj.grpc.helidon;
+
+import com.hedera.pbj.runtime.ServiceInterface;
+import io.helidon.builder.api.Option;
+import io.helidon.builder.api.Prototype;
+import io.helidon.webserver.spi.ProtocolConfig;
+
+@Prototype.Blueprint
+@Prototype.Configured
+@Prototype.Provides(ProtocolConfig.class)
+interface PbjConfigBlueprint extends ProtocolConfig {
+    /**
+     * Default maximum message size in bytes ({@value}).
+     *
+     * @see #maxMessageSize()
+     */
+    int DEFAULT_MAX_MESSAGE_SIZE = 1024*10; // 10KB
```

Comment:
It's best to include the units suffix to the name of the constant itself, e.g. `...SIZE_BYTES`. This way, when using this constant, it's always clear what the value actually represents.